catkin_ws/src/cucumber_msgs/include/cucumber_msgs/CucumberContainer.h

```diff
@@ -67,6 +68,15 @@ class CucumberContainer {
 		return msg;		 
 	}
 	
+	geometry_msgs::Pose createPose() {
+		geometry_msgs::Pose pose;
+		pose.position.x = x;
+		pose.position.y = y;
+		pose.position.z = z;
+		pose.orientation.w = 1.0;
+		return pose;
```

Comment:
Yes, although I have to admit that this value was arbitrarily chosen to make it a valid orientation.